flood_adapt/object_model/scenario.py

```diff
@@ -57,6 +58,23 @@ def save(self, filepath: Union[str, os.PathLike]):
     def run(self):
         """run direct impact models for the scenario"""
         self.init_object_model()
+        # start log file in scenario results folder
+        results_dir = self.database_input_path.parent.joinpath(
+            "output", "results", self.attrs.name
+        )
+        for parent in reversed(results_dir.parents):
+            if not parent.exists():
+                os.mkdir(parent)
+        if not results_dir.exists():
+            os.mkdir(results_dir)
+        # Initiate the logger for all the integrator scripts.
+        self.initiate_root_logger(results_dir.joinpath(f"{self.attrs.name}.log"))
+        version = "0.1.0"
+        logging.info(f"FloodAdapt version {version}")
```

Comment:
It shouldn't interfere since you are not including the overall logger (which is also not possible since this is the backend)